src/EthMultiVault.sol

```diff
@@ -278,6 +292,7 @@ contract EthMultiVault is IEthMultiVault, Initializable, ReentrancyGuardUpgradea
         timelocks[opHash].executed = true;
 
         emit AdminSet(admin, oldAdmin);
+        emit OperationExecuted(SET_ADMIN, data);
```

Comment:
3.17 - Issue an Event for Operation Execution

Here, we are now emitting an event for `OperationExecuted`